src/main/java/saber/commands/EventCommand.java

```diff
@@ -0,0 +1,69 @@
+package saber.commands;
+
+import saber.task.Event;
+import saber.exceptions.SaberTimeParserException;
+import saber.tasklist.TaskList;
+import saber.ui.EventUI;
+
+/**
+ * A class to encapsulate an EventCommand
+ */
+public class EventCommand extends SaberCommand {
+    private Event event;
+    private boolean isMissingDescription;
+    private boolean isMissingTime;
+    private boolean isParsingTimeError = false;
+
+    private EventUI eventUI = new EventUI();
+
+    /**
+     * A constructor for EventCommand
+     * @param eventTask the description of the event to be added
+     * @param eventTime the time of the event
+     * @param isMissingDescription whether the event description is missing in the command
+     * @param isMissingTime whether the event time is missing in the command
+     */
+    public EventCommand(String eventTask,
+                            String eventTime,
+                            boolean isMissingDescription,
+                            boolean isMissingTime) {
```

Comment:
Good line break in arguments to make it more readable 👍.
Indentation for wrapped lines should be 8 spaces more than the parent line though, so perhaps like this:
```suggestion
    public EventCommand(String eventTask,
            String eventTime,
            boolean isMissingDescription,
            boolean isMissingTime) {
```

Reply:
Ok